repository/c.json

```diff
@@ -3937,6 +3937,19 @@
 				}
 			]
 		},
+		{
+			"author": ["Tomas Barry"],
```

Comment:
I didn't know we supported arrays here, also doesn't seem necessary in this case.

Reply:
Addressed.